kegel_app/model/entities.py

```diff
@@ -22,11 +22,11 @@ class Units(enum.Enum):
     meters = 'm'
 
 
-class Thing(db.Model, mixins.DefaultMixin):
+class Thing(db.Model, mixins.InternalDefaultMixin):
     __tablename__ = 'things'
 
     name = db.Column(db.Unicode(50), nullable=False)
-    color = db.Column(db.Unicode)
+    color = db.Column(db.Unicode, info=dict(randomdata=lambda: Thing.random_color()))
```

Comment:
If you wanted to be a bit more "friendly" here, you could also setup your logic to detect a string and, if it's a string, assume it's a method on the class:

`randomdata='random_color'`

And your code will translate that into calling `cls.random_color()`.

Reply:
Implementation code for this on the issue: https://github.com/level12/keg-elements/issues/112#issuecomment-552954610